eng/CredScanSuppression.json

```diff
@@ -13,6 +13,7 @@
                 "Aa!1()-xyz",
                 ":code:`<pfx-file-password>`",
                 "000000000000000000000000000000000000000000000000000",
+                "fakekeyfakekeyfakekeyfakekeyfakekeyfakekeyA=",
```

Comment:
If possible it would be preferred to use an existing placeholder instead of adding more.